valohai_yaml/schema/toleration.yaml

```diff
@@ -0,0 +1,24 @@
+type: object
```

Comment:
I was considering a string based format `key1=value1:NoSchedule` used in **taint** CLI definitions but opted for similar object used in Kubernetes Pod specs (more familiar, probably)

aaand tolerations have further fields (at least `tolerationSeconds`) that _could_ be configured so leaving a door open for those too

Reply:
Definitely more readable this way!